src/diffusers/quantizers/bitsandbytes/utils.py

```diff
@@ -154,7 +154,7 @@ def replace_with_bnb_linear(model, modules_to_not_convert=None, current_key_name
 
 
 # Copied from PEFT: https://github.com/huggingface/peft/blob/47b3712898539569c02ec5b3ed4a6c36811331a1/src/peft/utils/integrations.py#L41
```

Comment:
Should this be updated to reflect the PEFT revision that also has these changes?

Reply:
Done. Thanks!